src/index.css

```diff
@@ -21,11 +25,25 @@ li {
   list-style: none;
 }
 
+.logo {
+  display: flex;
+  align-items: center;
+}
+
 header {
   display: flex;
   justify-content: space-between;
-  padding: 10px 1rem;
+  padding: 10px 4rem;
   align-items: center;
+  gap: 2rem;
+  background-color: #fff;
+}
+
+.header-logo {
+  font-size: 1.6rem;
```

Comment:
- [ ] These suggested changes are partially implemented by you but there is still some text that has wrong font. kindly check the fonts specification by clicking the text on the given design website. 
right now the `BOOKS and CATEGORIES` text next to heading in the navbar and text in the book names are not right.
your design
<img width="516" alt="Screenshot 2023-07-01 at 11 22 03 PM" src="https://github.com/derrick1451/bookstore/assets/96683843/b43eefc9-1f79-48cb-bcf9-85aaf9ab27f6">
given design
<img width="600" alt="Screenshot 2023-07-01 at 11 21 28 PM" src="https://github.com/derrick1451/bookstore/assets/96683843/3a9e7390-eec4-4fe6-946b-04df964e6a41">